app/models/provider_user.rb

```diff
@@ -36,10 +36,6 @@ def self.onboard!(dsi_user)
     end
   end
 
-  def can_view_safeguarding_information_for?(provider)
```

Comment:
:+1: As @mnacos has suggested, let's have a review of what permissions methods we've got and where they all live once the current PRs are in